From d924e7387dc6aaa755ebbe9fe2a016e0c4c0788e Mon Sep 17 00:00:00 2001
From: John Swinbank <swinbank@transientskp.org>
Date: Fri, 15 Jun 2012 11:22:12 +0200
Subject: [PATCH] Change hardcoded database host in gsm.py

---
 CEP/GSM/src/gsm.py |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CEP/GSM/src/gsm.py b/CEP/GSM/src/gsm.py
index e030546..bf2569f 100755
--- a/CEP/GSM/src/gsm.py
+++ b/CEP/GSM/src/gsm.py
@@ -42,12 +42,12 @@ def gsmMain (name, argv):
         theta = float(argv[5])
 
     #db_host = "ldb002"
-    db_host = "napels"
+    db_host = "heastrodb"
     db_dbase = "gsm"
     db_user = "gsm"
     #db_passwd = "msss"
-    db_passwd = "gsm"
-    db_port = 60000
+    db_passwd = "msss"
+    db_port = 52000
     db_autocommit = True
 
     try:
-- 
1.7.10

