From 14c6d3699ee3895b9d3fa8c126957f0e53a0bc2b Mon Sep 17 00:00:00 2001
From: John Swinbank <swinbank@transientskp.org>
Date: Thu, 10 Nov 2011 18:27:58 +0100
Subject: [PATCH] Add package LofarFT (awimager)

---
 CEP/Imager/CMakeLists.txt |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/CEP/Imager/CMakeLists.txt b/CEP/Imager/CMakeLists.txt
index f6999cf..d1676a1 100644
--- a/CEP/Imager/CMakeLists.txt
+++ b/CEP/Imager/CMakeLists.txt
@@ -4,3 +4,4 @@
 lofar_add_package(MWImager)
 lofar_add_package(LofarGridder)
 lofar_add_package(ASKAPsoft)
+lofar_add_package(LofarFT)
-- 
1.7.0.4

