From 7cb2a3eb39534136dd1b3568dcd5d3955bb4cda3 Mon Sep 17 00:00:00 2001
From: John Swinbank <swinbank@transientskp.org>
Date: Tue, 21 May 2013 12:53:24 +0200
Subject: [PATCH] Change hardcoded database host in gsm.py

---
 CEP/GSM/src/gsm.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CEP/GSM/src/gsm.py b/CEP/GSM/src/gsm.py
index c427575..e593821 100755
--- a/CEP/GSM/src/gsm.py
+++ b/CEP/GSM/src/gsm.py
@@ -47,13 +47,13 @@ def gsmMain (name, argv):
     if len(argv) > st+5:
         theta = float(argv[st+5])
 
-    db_host = "ldb002"
-    #db_host = "napels"
+    db_host = "heastrodb"
     db_dbase = "gsm"
     db_user = "gsm"
     db_passwd = "msss"
     #db_passwd = "gsm"
     db_port = 51000
+    db_port = 52000
     db_autocommit = True
 
     try:
-- 
1.7.10.4

