From 757ec722bd7f46fab57f451dd3af5c33087171b2 Mon Sep 17 00:00:00 2001
From: John Swinbank <swinbank@transientskp.org>
Date: Fri, 2 Dec 2011 18:16:47 +0100
Subject: [PATCH] Enable pipeline user to set niceness of remote jobs

---
 .../framework/lofarpipe/support/remotecommand.py   |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/CEP/Pipeline/framework/lofarpipe/support/remotecommand.py b/CEP/Pipeline/framework/lofarpipe/support/remotecommand.py
index 1d61993..10e3520 100644
--- a/CEP/Pipeline/framework/lofarpipe/support/remotecommand.py
+++ b/CEP/Pipeline/framework/lofarpipe/support/remotecommand.py
@@ -74,6 +74,14 @@ def run_remote_command(config, logger, host, command, env, arguments=None):
     except:
         method = None
 
+    try:
+        command = "nice --adjustment=%d %s" % (
+            int(config.get('cluster', 'nicelevel')),
+            command
+        )
+    except:
+        pass
+
     if method=="paramiko":
         try:
             key_filename = config.get('remote', 'key_filename')
-- 
1.7.7.4

