From 80cc254cd95b223bf529b35041f10e49851ae04a Mon Sep 17 00:00:00 2001
From: John Swinbank <j.swinbank@uva.nl>
Date: Fri, 20 Sep 2013 13:12:50 +0200
Subject: [PATCH] Correct exception spec

---
 casa/aipsdef.h | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/casa/aipsdef.h b/casa/aipsdef.h
index d806a5c..16b2a34 100644
--- a/casa/aipsdef.h
+++ b/casa/aipsdef.h
@@ -85,17 +85,17 @@ namespace std {};
 // SUN Native compiler has trouble with typedef inside class.
 // PGI compiler (QK_USER) on Cray XT3 needs throw specification in .cc file.
 #if defined(AIPS_SUN_NATIVE)
-#define WHATEVER_SUN_TYPEDEF(X) X::
-#define WHATEVER_TYPENAME
-#define WHATEVER_SUN_EXCEPTSPEC(X) throw(X)
+    #define WHATEVER_SUN_TYPEDEF(X) X::
+    #define WHATEVER_TYPENAME
+    #define WHATEVER_SUN_EXCEPTSPEC(X) throw(X)
 #else
-#define WHATEVER_SUN_TYPEDEF(X)
-#define WHATEVER_TYPENAME typename
-#if defined(AIPS_CRAY_PGI)
-#define WHATEVER_SUN_EXCEPTSPEC(X) throw(X)
-#else
-#define WHATEVER_SUN_EXCEPTSPEC(X)
-#endif
+    #define WHATEVER_SUN_TYPEDEF(X)
+    #define WHATEVER_TYPENAME typename
+    #if defined(AIPS_CRAY_PGI)
+        #define WHATEVER_SUN_EXCEPTSPEC(X) throw(X)
+    #else
+        #define WHATEVER_SUN_EXCEPTSPEC(X) throw(X)
+    #endif
 #endif
 
 #if defined(AIPS_USE_NEW_SGI) || defined(AIPS_GCC3) || defined(AIPS_GCC4) || defined(AIPS_CRAY_PGI)
-- 
1.8.4.rc3

