From 51c04252941761d746eb1522a523d18b9ef0fbfe Mon Sep 17 00:00:00 2001
From: John Swinbank <swinbank@transientskp.org>
Date: Tue, 2 Oct 2012 10:02:02 +0200
Subject: [PATCH] Fix compile with gcc-4.7

---
 casa/Arrays/IPosition.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/casa/Arrays/IPosition.h b/casa/Arrays/IPosition.h
index e6b7121..d35babd 100644
--- a/casa/Arrays/IPosition.h
+++ b/casa/Arrays/IPosition.h
@@ -34,6 +34,7 @@
 #include <casa/BasicSL/String.h>
 #include <vector>
 #include <cstddef>                  // for ptrdiff_t
+#include <unistd.h>
 
 namespace casa { //# NAMESPACE CASA - BEGIN
 
-- 
1.7.10.4

